migrate/migrate.go

```diff
@@ -100,6 +113,54 @@ func (m *Migrator) migrateToV0_1_0(id uuid.UUID) error {
 	return nil
 }
 
+// for migration to new seperate Contract struct re:flipfest-21
+func (m *Migrator) migrateToV0_8_0(id uuid.UUID) error {
+	proj := model.InternalProject{
+		ID: id,
+	}
+
+	var accounts []*model.InternalAccount
```

Comment:
instead of using `[]*model.InternalAccount` you could use a locally defined `[]*oldInternalAccount` this way you can still get rid of the old fields in `[]*model.InternalAccount`.

Reply:
yes, great suggestion. easier to maintain that way.